cradle-admin-tool-http/src/main/java/com/exactpro/th2/cradle/adm/http/PageManager.java

```diff
@@ -44,56 +55,78 @@ public class PageManager implements AutoCloseable, Runnable{
 
     public PageManager(
             CradleStorage storage,
+            boolean autoBooks,
             Map<String, AutoPageConfiguration> autoPages,
             int pageRecheckInterval,
             long pageActionRejectionThreshold
-    ) throws CradleStorageException {
+    ) {
         if (autoPages == null || autoPages.isEmpty()) {
-            logger.info("auto-page configuration is not provided, pages will not be generated automatically");
+            LOGGER.info("auto-page configuration is not provided, pages will not be generated automatically");
             this.storage = null;
             this.pageActionRejectionThreshold = 0;
             this.executorService = null;
             this.books = Collections.emptyMap();
             return;
         }
 
-        this.storage = storage;
-        this.pageActionRejectionThreshold = pageActionRejectionThreshold;
+        Map<String, String> normalisedBookName = autoPages.keySet().stream()
+                .collect(Collectors.toMap(
+                        PageManager::normaliseBookName,
+                        Function.identity()
+                ));
```

Comment:
What are we trying to do here? I see that we get a key set from the `autoPage` map and remap each key to its normalized version. Maybe we should get a stream of entries and remap the entries to normalized keys instead?
In that case, we don't need to use the original `autoPags` map when we initialize `books` field later in the constructor

Reply:
I would like to detect problem before interaction to cradle.

Reply:
Probably, I didn't explain myself clearly here. What I meant is that  we can collect the `Map<String, AutoPageConfiguration>` instead of collecting `Map<String, String>` and using the value later to get `AutoPageConfiguration` from `autoPages` map passed as a parameter into the constructor.
But it is all up to you

Reply:
I understood, thanks